pkg/kubectl/cmd/cp/cp.go

```diff
@@ -93,8 +97,13 @@ func NewCmdCp(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.C
 		Use:                   "cp <file-spec-src> <file-spec-dest>",
 		DisableFlagsInUseLine: true,
 		Short:                 i18n.T("Copy files and directories to and from containers."),
-		Long:                  "Copy files and directories to and from containers.",
-		Example:               cpExample,
+		Long: templates.LongDesc(`
+		Copy files and directories to and from containers.
+
+		Make local files containing ':' explicit by using relative or
```

Comment:
I find this a little confusing. I'm not sure how to fix it though.

Reply:
Yee, I kinda agree. I took the "idea" from the scp manpage, https://linux.die.net/man/1/scp, but couldn't find a better phrasing.. Hmm

Reply:
What do you think about this, from the manpage of scp

"Local file names can be made explicit using absolute or relative pathnames to avoid ~scp~ _kubectl cp_ treating file names containing ':' as pod specifiers."

Reply:
the wording is much better, I would definitely give an example right next to that sentence though, otherwise it still might not make sense. 

I read "relative or absolute" as complementary, hence not very useful.

But if that means it has to be either starting with / or ./ then I think an example might help.

Reply:
Good point! Updated the description and added a small example now.